cli/runtime/command/deprecation.go

```diff
@@ -22,16 +22,6 @@ func DeprecateCommandWithAlternative(cmd *cobra.Command, removalVersion, alterna
 	cmd.Deprecated = msg
 }
 
-// DeprecateFlag marks the flag as deprecated and hidden with a deprecation message.
-func DeprecateFlag(cmd *cobra.Command, flag, removalVersion string) {
-	f := cmd.Flags().Lookup(flag)
-	if f != nil {
-		msg := fmt.Sprintf("will be removed in version %q.", removalVersion)
-		f.Deprecated = msg
-		f.Hidden = true
-	}
-}
```

Comment:
Is there any reason behind remove this function?

Reply:
I think it will be helpful in future. So, would be nice to keep this function around.

Reply:
That works. I'll re-add it, and add unit tests for it. Thanks for the feedback @anujc25 !